test_conformance/math_brute_force/binary_half.cpp

```diff
@@ -312,9 +317,9 @@ cl_int TestHalf(cl_uint job_id, cl_uint thread_id, void *data)
         s2[j] = cl_half_to_float(p2[j]);
         if (isNextafter)
             r[j] = cl_half_from_float(reference_nextafterh(s[j], s2[j]),
```

Comment:
The changes look reasonable to me.

However this affects all binary fp16 operations as far as I can tell, not only `fdim`?  It would be good to make that clear in the commit message and description; right now it suggests the change would only affect `fdim` which is not the case.